app/build.gradle

```diff
@@ -111,6 +111,7 @@ android {
 
     lintOptions {
         disable 'InvalidPackage'
+        abortOnError false
```

Comment:
It's not bad solution 😃 
To be honest, I don't want to change this option 🙇 
Because sometimes this option prevents critical error in the build process.
Can we solve this by fixing `multiline-collapsingtoolbar`?